hugegraph-it/hg-it-minicluster/src/main/java/org/apache/hugegraph/it/node/BaseNodeWrapper.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.hugegraph.it.node;
+
+public interface BaseNodeWrapper {
+
+    void createNodeDir();
```

Comment:
i can't think these method are needed
sometimes, i am confused, what is  the responsibility of the wrapper class's;
if i can't understand wrong, i think the createNodeDir or createLogDir, deleteDir method are the spec impl logic, their should been call in the wrap inner in the different lifecycle. don't need expose for other class call

Reply:
should the log dir been delete with external tools?